### PR TITLE
Initial support for logout from both client and server.

### PR DESCRIPTION
**Pull request by jasoncouture**
TODO: Event system will need to be reworked to enforce logout delay.

### CODE REVIEW

File: Source/NexusForever.WorldServer/Network/Message/Handler/ClientLogoutHandler.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+using NexusForever.Shared.Game.Events;
+using NexusForever.Shared.Network.Message;
+using NexusForever.WorldServer.Network.Message.Model;
+using NLog;
+
+namespace NexusForever.WorldServer.Network.Message.Handler
+{
+
+    public static class ClientLogoutHandler
```
Comment (MaxtorCoder) [inline Source/NexusForever.WorldServer/Network/Message/Handler/ClientLogoutHandler.cs]: You should make this in the CharacterHandler. 😄

Comment (Rawaho) [reply Source/NexusForever.WorldServer/Network/Message/Handler/ClientLogoutHandler.cs]: I do agree with this, I doubt we need an entire handler class for a single packet

**Comment by Rawaho** [comment]
I think it would be best until the event system is reworked to remove the event entirely since it will probably need to be changed when the system is updated anyway.

File: Source/NexusForever.WorldServer/Network/Message/Handler/ClientLogoutHandler.cs (hunk 1 above)
Comment (jasoncouture) [reply Source/NexusForever.WorldServer/Network/Message/Handler/ClientLogoutHandler.cs]: I made both requested changes.

Comment (MaxtorCoder) [reply Source/NexusForever.WorldServer/Network/Message/Handler/ClientLogoutHandler.cs]: Thanks you!

**Pull request merged**